package.json

```diff
@@ -10,7 +10,7 @@
     "Lob.com",
     "printing"
   ],
-  "version": "3.9.0",
+  "version": "3.10.0",
```

Comment:
You shouldn't update this version manually. Our release script will automatically update it for you.